seed/php-sdk/alias-extends/src/Exceptions/SeedApiException.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Seed\Exceptions;
+
+use Throwable;
+
+/**
+ * This exception type will be thrown for any non-2XX API responses.
+ */
+class SeedApiException extends SeedException
+{
+    /**
+     * @var string $body
+     */
+    private string $body;
+
+    /**
+     * @param string $message
+     * @param int $statusCode
+     * @param string $body
+     * @param ?Throwable $previous
+     */
+    public function __construct(
+        string $message,
+        int $statusCode,
+        string $body,
```

Comment:
in c# we made this an `object` (equivalent in php would be `mixed`) so that when we extend the `ApiException` to make the specific error types, we can make the body a specific class

i think we'll want to go ahead and do this here so that people aren't depending on the body being a string, although I haven't played around with this in PHP at all. happy to help with that, but wanted to check in with your thinking here first

Reply:
Great shout out - I didn't actually realize that's why we modeled this as `object` in C#. We should definitely do the same thing here, so let's go with `mixed`.